web/ui/react-app/src/utils/index.ts

```diff
@@ -197,3 +197,7 @@ export const toQueryString = ({ key, options }: PanelMeta) => {
 export const encodePanelOptionsToQueryString = (panels: PanelMeta[]) => {
   return `?${panels.map(toQueryString).join('&')}`;
 };
+
+export const createExpressionLink = (expr: string) => {
```

Comment:
Please pass as second argument the ```pathPrefix``` prop and remove ```../``` - this should be passed from outside as well